### PR TITLE
Add disable_sector management command

### PR DESCRIPTION
**Pull request by abbas123456**
### Description of change

This adds a disable_sector management command to allow Sectors to be disabled if they are no longer required.

This will allow orphaned Sectors to be tidied up after the update_sector_segment(#2899), create_sector(#2907), and update_sector_parent (#2920) commands are run.

### Checklist

* [x] If this is a releasable change, has a news fragment been added?

  <details>
  <summary>Explanation</summary>
  
  A news fragment is required for any releasable change (i.e. code that runs in or affects production) so that a corresponding changelog entry is added when releasing.
  
  Check [changelog/README.md](https://github.com/uktrade/data-hub-api/blob/master/changelog/README.md) for instructions.
  
  </details>
  
* [x] Has this branch been rebased on top of the current `develop` branch?

  <details>
  <summary>Explanation</summary>
  
  The branch should not be stale or have conflicts at the time reviews are requested.
  
  </details>

* [x] Is the CircleCI build passing?

### General points

<details>
<summary><strong>Other things to check</strong></summary><p></p>

* Make sure `fixtures/test_data.yaml` is maintained when updating models
* Consider the admin site when making changes to models
* Use select-/prefetch-related field lists in views and search apps, and update them when fields are added
* Make sure the README is updated e.g. when adding new environment variables

</details>

See [docs/CONTRIBUTING.md](https://github.com/uktrade/data-hub-api/blob/develop/docs/CONTRIBUTING.md) for more guidelines.

### CODE REVIEW

File: datahub/dbmaintenance/management/commands/disable_sector.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    def _process_row(self, row, simulate=False, **options):
+        """Process one single row."""
+        pk = parse_uuid(row['id'])
+        sector = Sector.objects.get(pk=pk)
+
+        if sector.get_children():
+            logger.warning(f'Not disabling sector {sector} as it has children')
+            return
+
+        sector.disabled_on = now()
```
Comment (marcuspp) [inline datahub/dbmaintenance/management/commands/disable_sector.py:26]: To make the script re-run able and to produce the same result each time maybe this should come from the CSV file or be a parameter passed in when running the script.

File: datahub/dbmaintenance/management/commands/disable_sector.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+class Command(CSVBaseCommand):
+    """Command to disable a Sector."""
+
+    def _process_row(self, row, simulate=False, **options):
+        """Process one single row."""
+        pk = parse_uuid(row['id'])
+        sector = Sector.objects.get(pk=pk)
+
+        if sector.get_children():
```
Comment (marcuspp) [inline datahub/dbmaintenance/management/commands/disable_sector.py:22]: Would this mean we could not be able to disable both a parent and child? Say if `A : B` and `A : B : C` where now both obsolete? (I will have a look at the data to see if this occurs at all)

Comment (abbas123456) [reply datahub/dbmaintenance/management/commands/disable_sector.py]: Will disable all descendants too

File: datahub/dbmaintenance/management/commands/disable_sector.py (hunk 1 above)
Comment (abbas123456) [reply datahub/dbmaintenance/management/commands/disable_sector.py]: Will take disabled_on from the csv

**Comment by abbas123456** [comment]
Disabled sectors still appear on the front end so a command to delete sectors will be written instead.

**Pull request closed without merging**